app/views/kaui/refunds/_form.html.erb

```diff
@@ -213,6 +258,13 @@
             return this.id.match(/cb_adj_/);
         }).click(onClickInvoiceItemAdjustment);
 
+        /*
+        * Attach handler onClickInvoiceItemsSelectAll for selectall checkbox
+        */
+        $('input').filter(function() {
```

Comment:
Since there is only one Select All checkbox (pls correct if I am wrong) with known id (`selectall` or `selectAll`), can we use it directly like
`$('#selectAll').click(onClickInvoiceItemsSelectAll);`?

Reply:
Thanks for pointing out we have update the event binding to `$('#selectAll').click(onClickInvoiceItemsSelectAll);`